scripts/compare_strategies.py

```
@@ -0,0 +1,334 @@
+            "single_hop": round(scores_b["single_hop"], 1),
+            "temporal": round(scores_b["temporal"], 1),
+            "multi_hop": round(scores_b["multi_hop"], 1),
+            "open_domain": round(scores_b["open_domain"], 1),
+            "adversarial": round(scores_b["adversarial"], 1),
+            "evidence_recall": round(scores_b["evidence_recall"], 1),
+            "avg_query_ms": round(scores_b["avg_query_ms"], 1),
+            "p95_query_ms": round(scores_b["p95_query_ms"], 1),
+            "embedder": report["embedder"],
+            "commit": "",
+            "date": datetime.now().strftime("%Y-%m-%d"),
+            "notes": f"auto-updated via compare_strategies.py",
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)